portal-kernel/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: ${manifest.bundle.name}
 Bundle-SymbolicName: ${manifest.bundle.symbolic.name}
-Bundle-Version: 5.5.0
```

Comment:
@brianchandotcom, not sure if this is really needed or if we could change this in a different way :(

Reply:
Ah, I think we mixed in setting the theme by default and just commiting the theme...